packages/altive_lints/lib/src/lints/prefer_space_between_elements.dart

```diff
@@ -69,23 +69,23 @@ class PreferSpaceBetweenElements extends DartLintRule {
             nextMember is MethodDeclaration &&
             nextMember.name.lexeme == 'build') {
           if (!_hasBlankLineBetween(currentMember, nextMember, lineInfo)) {
-            reporter.reportErrorForNode(code, nextMember);
+            reporter.atNode(nextMember, code);
```

Comment:
![badge](https://img.shields.io/badge/review-imo-important) 
It seems that a similar analyze error is occurring here as well!  
Please revise it to pass `_code`.

Reply:
It occurs with analyzer v6.8.0 or higher.

Thanks! I'll make sure to use it _code 👍